src/components/Dao/ProposalSchemeCard.tsx

```diff
@@ -54,7 +54,7 @@ const ProposalSchemeCard = (props: IProps) => {
         <div>
           <b>{schemeState.numberOfBoostedProposals}</b> <span>Boosted</span> <b>{schemeState.numberOfPreBoostedProposals}</b> <span>Pending Boosting</span> <b>{schemeState.numberOfQueuedProposals}</b> <span>Regular</span>
         </div>
-        {proposals.length === 0 ?
+        {numProposals === 0 ?
```

Comment:
how this is related ?

Reply:
Reviewing this I find I fixed it wrong.  So I've undone this change but added a different change on line 101 of this file.  The problem is that the list of boosted proposals was not going to be responsive to new boosted or unboosted proposals.